setup.py

```diff
@@ -34,10 +35,28 @@ def __init__(self, attrs=None):
         self.ext_modules = [Extension('h3c', [])]
 
 
+try:
+    import pypandoc
+    long_description = pypandoc.convert('README.md', 'rst')
+    blacklist = [
+        'Build Status', 'Latest Version', 'Documentation Status',
+        'travis-ci.org'
+    ]
+    long_description = '\n'.join([
+        line for line in long_description.split('\n')
+        if not any(patt in line for patt in blacklist)
```

Comment:
Agreed, that seems a lot easier.

Reply:
GitHub deals with a number of formats. Not all of them are feature complete. AsciiDoc is supported, but including files into the document using the `include` directive is not, seeing as it can be used to cause infinite recursion attacks on the server.